# HG changeset patch
# User Ben Abbott  <bpabbott@mac.com>
# Date 1292547433 18000
#      Thu Dec 16 19:57:13 2010 -0500
# Node ID 572a318eb715ae50b7b34b8302cb905f8a91bc39
# Parent  d67d44f47949289469c3a9ec7e4c9d3edbf546df
unimplemented.m: Remove isprop() form list of unimplemented functions.

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,10 +1,12 @@
 2010-12-16  Ben Abbott <bpabbott@mac.com>
 
+	* miscellaneous/unimplemented.m: Remove isprop() form list of
+	unimplemented functions.
 	* plot/isprop.m: New function.
 
 2010-12-16  Olaf Till <olaf.till@uni-jena.de>
 
 	* optimization/sqp.m: Fix indexing error in sqp bounds selection. 
 
 2010-12-14  Doug Stewart <doug.dastew@gmail.com>
 
diff --git a/scripts/miscellaneous/unimplemented.m b/scripts/miscellaneous/unimplemented.m
--- a/scripts/miscellaneous/unimplemented.m
+++ b/scripts/miscellaneous/unimplemented.m
@@ -24,17 +24,17 @@
 
 function txt = unimplemented (fcn)
 
   is_matlab_function = true;
 
   ## Some smarter cases, add more as needed.
   switch (fcn)
 
-    case "quad2d"
+  case "quad2d"
     txt = ["quad2d is not implemented.  Consider using dblquad."];
 
   case "gsvd"
     txt = ["gsvd is not currently part of Octave.  See the linear-algebra",... 
     "package at @url{http://octave.sf.net/linear-algebra/}."];
 
   case "linprog"
     txt = ["Octave does not currently provide linprog.  ",...
@@ -232,17 +232,16 @@ function list = missing_functions ()
   "instrfind", 
   "instrfindall", 
   "interpstreamspeed", 
   "iscom", 
   "isinterface", 
   "isjava", 
   "isocaps", 
   "ispref", 
-  "isprop", 
   "isstudent", 
   "javaArray", 
   "javaMethod", 
   "javaMethodEDT", 
   "javaObject", 
   "javaObjectEDT", 
   "javaaddpath", 
   "javachk", 
